tests/combo/converter/combo_atomeseUTest.cxxtest

```diff
@@ -36,70 +36,70 @@ using namespace opencog;
 class combo_atomeseUTest : public CxxTest::TestSuite{
 
 public:
-    void test_atomese_combo_predicate(void){
-        combo::combo_tree tr = combo::str2combo_tree("and(or($1 $2) not($3))", {});
+	void test_atomese_combo_predicate(void){
```

Comment:
@behailu04, sorry to bother you with another minor formatting detail, we usually put a space (or a new line) between `)` and `{`. I don't think it's specified in the dev standard, although the examples given do follow that style.

Reply:
Ok Thanks nil.